### ISSUE TITLE
[Accessibility] Missing aria-label for expand option in the drawer's header

### ISSUE THREAD

**Issue by srenault-meeds** [Meeds-io/meeds#3693]
### Problem Statement

Given a drawer
Then the expand option is displayed as expected at the header of the drawer

Problem1: Aria-label is missing to make it readable by screenreaders

<img width="1327" height="136" alt="Image" src="https://github.com/user-attachments/assets/3b8f5fa0-64ac-4ffc-9685-16cde7a21275" />

### Bug

**Expected**
Add aria-label for this option (same as title attribute)
